tests/test_real_projects.py

```diff
@@ -183,6 +187,7 @@ def test_real_project(request, project, experiment):
 
     print(f"Testing real project {project.name!r} under {project_dir}")
     print(f"Project description: {project.description}")
+    print(f"Virtual environment of the project {project.name!r} under {venv_dir}")
```

Comment:
The project name is already mentioned two lines above.

```suggestion
    print(f"Virtual environment under {venv_dir}")
```